kuma/javascript/src/payments/subscription-form.jsx

```diff
@@ -0,0 +1,107 @@
+    const locale = getLocale();
+    const subscriptionFormRef = useRef(null);
+    const [subscribeFormDisabled, setSubscribeFormDisabled] = useState(false);
+    const [subscribeButtonEnabled, setSubscribeButtonEnabled] = useState(false);
+
+    const toggleButton = () => {
+        setSubscribeButtonEnabled(!subscribeButtonEnabled);
+    };
+
+    /**
+     * Opens Stripe modal allowing a user to complete their subscription.
+     * @param {Object} event - The form submit event
+     */
+    const submit = event => {
+        event.preventDefault();
+
+        const subscriptionForm = subscriptionFormRef.current;
+        if (subscriptionForm) {
+            const formData = new FormData(subscriptionForm);
+            setSubscribeFormDisabled(true);
+
+            const stripeHandler = window.StripeCheckout.configure({
+                key: window.mdn.stripePublicKey,
+                locale: 'auto',
+                name: 'MDN Web Docs',
+                zipCode: true,
+                currency: 'usd',
+                amount: 500,
+                email: '',
+                token: function(response) {
+                    formData.set('stripe_token', response.id);
+                    subscriptionForm.submit();
+                },
+                closed: function() {
+                    if (!formData.get('stripe_token')) {
+                        setSubscribeFormDisabled(false);
+                    }
+                }
+            });
+
+            stripeHandler.open();
+        }
+    };
+
+    return (
+        <div className="subscriptions-form">
+            <header className="subscriptions-form-header">
+                <h2>
+                    <Interpolated
+                        id={gettext('$5 <perMontSub />')}
+                        perMontSub={<sub>{gettext('/mo')}</sub>}
+                    />
+                </h2>
+            </header>
+            <form
+                ref={subscriptionFormRef}
+                name="subscription-form"
+                method="post"
+                onSubmit={submit}
+                disabled={subscribeFormDisabled ? 'disabled' : ''}
+            >
+                <label className="payment-opt-in">
+                    <input
+                        type="checkbox"
+                        required="required"
+                        onClick={toggleButton}
+                    />
+                    <small>
+                        <Interpolated
+                            id={gettext(
+                                'By clicking this button, I authorize Mozilla to charge this payment method each month, according to the <paymentTermsLink />, until I cancel my subscription.'
+                            )}
+                            paymentTermsLink={
+                                <a
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                    href={`/${locale}/payments/`}
```

Comment:
Shouldn't that be 
```suggestion
                                    href={`/${locale}/payments/terms/`}
```
Which is still a Wiki only page.

Reply:
Could very well be, this is what it is currently on edit profile. @Gregoor thoughts?

Reply:
Nope, nope, my bad. Updating.